include/net/nrf_cloud_pgps.h

```diff
@@ -238,6 +240,51 @@ int nrf_cloud_pgps_request(const struct gps_pgps_request *request);
 int nrf_cloud_pgps_request_all(void);
 #endif /* CONFIG_NRF_CLOUD_MQTT */
 
+#if defined(CONFIG_NRF_CLOUD_PGPS_DOWNLOAD_TRANSPORT_CUSTOM)
+/**@brief When you use a custom transport mechanism for downloading prediction
+ * data, call this function when your pgps_event_handler_t receives a
+ * PGPS_EVT_REQUEST event, after your handler has sent the P-GPS request to the
+ * cloud and before the first packet is received. It resets the internal state
+ * of the library and prepares the flash storage to receive incoming data.
+ *
+ * Call the nrf_cloud_pgps_finish_update() function if this function returns an error.
+ *
+ * @retval 0 Ready for updating.
+ * @retval -ENOMEM No free space in P-GPS flash storage area.
+ * @retval -EBUSY P-GPS download already in progress.
+ * @retval -ENODEV Flash device is not ready.
+ * @retval -EFAULT Error initializing flash stream or on buffered write to it.
```

Comment:
`Error initializing or writing to flash`?

Reply:
I traced through all the paths that generate errors, and there are two places that return -EFAULT. I guess I could just say "Error accessing flash stream."

Reply:
Any comment?

Reply:
yes this change is good